### PR TITLE
gui: fix go-1.10 compatibility

### PR DESCRIPTION
**Pull request by dawidd6**

### CODE REVIEW

File: pkg/gui/recording.go [hunk 1]
```diff
@@ -21,7 +21,7 @@ func recordEventsTo() string {
 }
 
 func (gui *Gui) timeSinceStart() int64 {
-	return time.Since(gui.StartTime).Milliseconds()
+	return time.Since(gui.StartTime).Nanoseconds() / 1e6
```
Comment (jesseduffield) [inline pkg/gui/recording.go:24]: shouldn't this be `* 1e6`

**Review by jesseduffield — CHANGES_REQUESTED** [review]
one question

File: pkg/gui/recording.go (hunk 1 above)
Comment (dawidd6) [reply pkg/gui/recording.go:24]: I got this from here https://golang.org/src/time/time.go?s=24262:24300#L725

Also nano is less than mili

Comment (jesseduffield) [reply pkg/gui/recording.go:24]: goes to show what happens when you haven't touched maths since university

**Review by jesseduffield — APPROVED** [review]
You win this time @dawidd6, but I'll be back

**Pull request merged**